simulation_parameters/Constants.cs

```diff
@@ -427,9 +430,10 @@ public static string Version
         get
         {
             var assembly = Assembly.GetExecutingAssembly();
-            Version version = assembly.GetName().Version;
-            var versionSuffix = (AssemblyInformationalVersionAttribute[])assembly.
-                GetCustomAttributes(typeof(AssemblyInformationalVersionAttribute), false);
+            var version = assembly.GetName().Version;
```

Comment:
Just a nitpick, but why can't the variable type just be specified in this scenario? Is it for better organization?

Reply:
This is what the code cleanup tool wanted to do, and I accepted. I think I turned this option off for further use as I couldn't tweak when to use var exactly to my liking.